scala/spark-wordcount/src/main/scala/com/example/bigtable/spark/wordcount/WordCount.scala

```diff
@@ -127,15 +127,17 @@ object WordCount {
     job.setOutputFormatClass(classOf[TableOutputFormat[ImmutableBytesWritable]])
     conf = job.getConfiguration()
 
-    val wordCounts = sc.textFile(fileName).
-      flatMap(_.split(" ")).
-      filter(_ != "").map((_, 1)).
-      reduceByKey((a, b) => a + b)
-      .map({
-        case (word, count) =>
-          (null, new Put(Bytes.toBytes(word))
-            .addColumn(ColumnFamilyBytes, ColumnNameBytes, Bytes.toBytes(count)))
-      })
+    val wordCounts = sc
+      .textFile(fileName)
+      .flatMap(_.split("\\s+"))
+      .filter(!_.isEmpty)
+      .map { word => (word, 1) }
+      .reduceByKey(_ + _)
+      .map { case (word, count) =>
+        val put = new Put(Bytes.toBytes(word))
+          .addColumn(ColumnFamilyBytes, ColumnNameBytes, Bytes.toBytes(count))
+        (null, put)
```

Comment:
Please add a comment explaining what the null is

Reply:
Honestly? I've got no clue why it's always been `null` 😎

Reply:
Please add the comment something along the lines of 

The underlying writer ignores keys, only the value matter here.

Reply:
done

Reply:
I think you might've forgotten to push, I dont see it :(

Reply:
please check it out now. Sorry.